go/keybase/main.go

```diff
@@ -115,6 +115,10 @@ func mainInner(g *libkb.GlobalContext) error {
 		return err
 	}
 
+	if err = configurePath(g, cl); err != nil {
```

Comment:
this happens after FixVersion right?  that is **CRUCIAL**.  can you make a comment about it?